# Time-varying Linear Gaussian controller
# Keep T copies of each
std_msgs/Float64MultiArray K_t
std_msgs/Float64MultiArray k_t
std_msgs/Float64MultiArray x_hat_t
std_msgs/Float64MultiArray u_hat_t
std_msgs/Float64MultiArray PSig_t
